project/Deps.scala

```diff
@@ -12,11 +12,11 @@ object Deps {
       .exclude("com.twitter", "finagle-zipkin_2.12")
 
   def twitterUtil(mod: String) =
-    "com.twitter" %% s"util-$mod" % "6.45.0"
+    "com.twitter" %% s"util-$mod" % "7.0.0"
 
   // networking
   def finagle(mod: String) =
-    "com.twitter" %% s"finagle-$mod" % "6.45.0"
+    "com.twitter" %% s"finagle-$mod" % "7.0.0"
 
   def netty4(mod: String) =
     "io.netty" % s"netty-$mod" % "4.1.10.Final"
```

Comment:
Can you also bump this netty dependency to match Finagle 7's version? It should be "4.1.12.Final".